listenstore/influx_listenstore.py

```diff
@@ -324,12 +135,12 @@ def get_timestamps_for_user(self, user_name):
         else:
             query = """SELECT first(artist_msid)
                          FROM listen
-                        WHERE user_name = '""" + user_name + "'"
+                        WHERE user_name = '""" + user_name + """'"""
             min_ts = self._select_single_timestamp(query)
 
             query = """SELECT last(artist_msid)
                          FROM listen
-                        WHERE user_name = '""" + user_name + "'"
+                        WHERE user_name = '""" + user_name + """'"""
```

Comment:
Did this because sublime text's syntax highlighting got confused somehow with the previous version